.markdownlint.yaml

```diff
@@ -1,3 +1,2 @@
 ---
-MD013:
-  line_length: 120
+MD013: false
```

Comment:
Good that you are bringing this up again and I do also have my doubt whether a 120 char limit is reasonable for MD013. Therfore, I started to research some other repositories.
Repos like [azure-docs](https://github.com/MicrosoftDocs/azure-docs), [freeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp) have it deactiavted.
Other's set more reasonable limits like 400 (see [chef-web-docs](https://github.com/chef/chef-web-docs), or 2000 [rails](https://github.com/rails/rails/blob/bab4aa7cb25112846e04cea907361a1de0f126ef/.mdlrc.rb#L20).

My suggestion is not to fully deactivate. My preference right now lies at 400 characters (since a paragraph itself should not be more than a couple of sentences). Otherwise, I suggest adopting the setting from the, for example, the rails repository, for the following reasons:
- fully deactivating will be more difficult to revert,
- accessibility reasons (i.e., screen readers),
- easier version control.